Userland/Applications/Spreadsheet/HelpWindow.cpp

```diff
@@ -74,7 +74,8 @@ HelpWindow::HelpWindow(GUI::Window* parent)
     auto& splitter = widget.add<GUI::HorizontalSplitter>();
     auto& left_frame = splitter.add<GUI::Frame>();
     left_frame.set_layout<GUI::VerticalBoxLayout>();
-    left_frame.set_fixed_width(100);
+    //FIXME: Get rid of the magic number, dynamically calculate initial size based on left frame contents
```

Comment:
CI is unhappy that there's no space between the `//` and `FIXME` here.

Reply:
Thanks for explaining that, I couldn't figure out what I'd missed. I've setup clang-format and the commit hooks, and have amended this PR.